libs/blocks/merch-card/merch-card.css

```
@@ -103,6 +111,16 @@
   border-radius: 16px;
 }
 
+.merch-card.special-offers.full-width {
```

Just a thought.
Is there any reason to scope these too .special-offers / would other types benefit from having them? 
```suggestion
.merch-card.full-width {
```
Below desktop VP the `--grid-container-width` is 83%. Is that the intended view for tablet? I see this goes full-width on mobile and desktop but the tablet does not? In the jira ticket it says to be a section property. Is this right. There is a .con-block.full-width style that we use for our ups grid override, but I know the cards do their own thing. 
> Use/develop “full width” instead of “one-up” for full size cards (this will be a section property)

---

-As per Consonant guide, only the Special Offers card will use `full-width`. 
-No, 83% is not intended for tablet, this is a good catch. If you think it's ok, I can use 1200px hardcoded which is the default for containers in Milo. I have tested with this value and the card adjusts accordingly as expected in mobile/tablet.
-About this being a section metadata property, I have asked Fred (GWP) to update the description since we have agreed to keep it as a block property instead to avoid impacting other blocks.

---

ticket description has been updated, full-width is expected to be a block property instead of section-metadata